packages/react/src/components/Forms/FileUploader/index.spec.tsx

```diff
@@ -0,0 +1,557 @@
+      expect(document.querySelector(".material-icons")?.textContent).toContain(
+        "done"
+      );
+    });
+    it("should be in state=error", async () => {
+      render(
+        <CunninghamProvider>
+          <FileUploader
+            state="error"
+            bigText="Error file is too big"
+            multiple={true}
+          />
+        </CunninghamProvider>
+      );
+
+      // Big text is shown is file is not selected.
+      expect(
+        document.querySelector(".c__file-uploader--error")
+      ).toBeInTheDocument();
+      screen.getByText("Error file is too big");
+
+      const user = userEvent.setup();
+      const input: HTMLInputElement = document.querySelector("input")!;
+      const files = [
+        new File(["hello"], "hello.png", { type: "image/png" }),
+        new File(["foo"], "foo.png", { type: "image/png" }),
+      ];
+
+      await act(async () => {
+        await user.upload(input, files);
+      });
+
+      expectFiles([
+        { name: "hello.png", specs: "5 Bytes" },
+        { name: "foo.png", specs: "3 Bytes" },
+      ]);
+
+      // The error is still displayed if files are selected.
+      expect(
+        document.querySelector(".c__file-uploader--error")
+      ).toBeInTheDocument();
+      screen.getByText("Error file is too big");
+    });
+    it("should display text", async () => {
+      render(
+        <CunninghamProvider>
+          <FileUploader
+            text="JPG, PNG or GIF - Max file size 2MB"
+            multiple={true}
+          />
+        </CunninghamProvider>
+      );
+      screen.getByText("JPG, PNG or GIF - Max file size 2MB");
+    });
+    it("can be reset in a controlled way and triggers onChange", async () => {
+      const Wrapper = () => {
+        const ref = useRef<FileUploaderRefType>(null);
+        const [value, setValue] = useState<File[]>([]);
+        return (
+          <CunninghamProvider>
+            <div>
+              <div>Value: {value.map((file) => file.name).join(", ")}|</div>
+              <FileUploader
+                onChange={(e) => setValue(e.target.value)}
+                ref={ref}
+              />
+              <Button onClick={() => ref.current?.reset()}>Reset</Button>
+            </div>
+          </CunninghamProvider>
+        );
+      };
+      render(<Wrapper />);
+      const user = userEvent.setup();
+      const input: HTMLInputElement =
+        screen.getByLabelText(/Drag and drop or /);
+      const file = new File(["hello"], "hello.png", { type: "image/png" });
+
+      // No file is selected.
+      screen.getByText("Value: |");
+
+      // Upload a file.
+      await act(async () => {
+        await user.upload(input, file);
+      });
+      screen.getByText("Value: hello.png|");
+
+      // Reset the file.
+      const resetButton = screen.getByRole("button", { name: "Reset" });
+      await act(async () => {
+        await user.click(resetButton);
+      });
+      screen.getByText("Value: |");
+    });
```

Comment:
You are in the `Multiple` test suite, should be moved into `Mono` or a dedicated `Controlled` test suite ?

Reply:
Just a miss, forgot to add multiple and multiples files